org.firestormviewer.FirestormViewer.json

```diff
@@ -0,0 +1,89 @@
+{
+  "app-id":"org.firestormviewer.FirestormViewer",
+  "branch":"stable",
+  "runtime":"org.freedesktop.Platform",
+  "runtime-version":"19.08",
+  "sdk":"org.freedesktop.Sdk",
+  "command":"firestorm-viewer",
+  "separate-locales":false,
+  "finish-args":[
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--share=network",
+    "--allow=multiarch",
+    "--device=dri",
+    "--filesystem=home",
+    "--talk-name=org.freedesktop.Notifications",
+    "--own-name=com.secondlife.ViewerAppAPIService",
+    "--filesystem=xdg-run/app/com.discordapp.Discord:create"
+  ],
+  "cleanup":[
+    "/share/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/lib/pkgconfig",
+    "/lib/debug"
+  ],
+  "build-options":{
+    "strip":true
+  },
+  "add-extensions":{
+    "org.freedesktop.Platform.Compat.i386":{
+      "directory":"lib/i386-linux-gnu",
+      "version":"19.08"
+    }
+  },
+  "sdk-extensions":[
+    "org.freedesktop.Sdk.Compat.i386",
+    "org.freedesktop.Sdk.Extension.toolchain-i386"
+  ],
+  "modules":[
+    "shared-modules/glu/glu-9.json",
+    "shared-modules/gtk2/gtk2.json",
+    "shared-modules/dbus-glib/dbus-glib-0.110.json",
+    "modules/wine.json",
+    "modules/libnotify.json",
+    "modules/gamemode.json",
+    {
+      "name":"firestorm",
+      "buildsystem":"simple",
+      "cleanup":"viewer.tar.xz",
+      "sources":[
+        {
```

Comment:
Does the license allow redistribution?

Reply:
There's two libraries in there, Kakadu and Vivox, which have licenses that don't allow redistribution.

I'll swap it out of the build stage and into an extra-data stage. Eventually it should be possible to build without Kakadu and fetch Vivox as an extra.

Reply:
I've swapped this to extra-data.

The symbolic was under a CC-like license and it's hard to attribute it, so it's been removed. The main icon is under LGPL (need to triple check) and that's now fetched from the source code.